### ISSUE TITLE
Loss for FastDeepMatting model

### ISSUE THREAD

**Issue by rose-jinyang** [ItchyHiker/Hair_Segmentation_Keras#3]
Hello
How are you?
Thanks for contributing to this project.
I have a question.
In the fast deep matting paper (https://arxiv.org/pdf/1707.08289.pdf), the author used loss = L_alpha + L_color.
We can see this loss implementation by PyTorch in https://github.com/lizhengwei1992/mobile_phone_human_matting.
But I can NOT see this loss in this repo.
Could u let me know about your loss implementation in detail?
Thanks.
